### PR TITLE
User must have namely field to import candidates

### PR DESCRIPTION
**Pull request by tabfugnic**
* Change copy for missing namely field to be more strict
* Refactor connections on dashboard into partial
* Build presenters for connections to reduce duplication

### CODE REVIEW

File: app/presenters/icims/connection_presenter.rb [hunk 1]
```diff
@@ -0,0 +1,29 @@
+module Icims
+  class ConnectionPresenter < SimpleDelegator
+    include ActiveModel::Conversion
+
+    def initialize(connection)
+      super(connection)
```
Comment (dgalarza) [inline app/presenters/icims/connection_presenter.rb]: What's the reasoning behind having an initializer that just calls super?

File: app/presenters/jobvite/connection_presenter.rb [hunk 2]
```diff
@@ -0,0 +1,29 @@
+module Jobvite
+  class ConnectionPresenter < SimpleDelegator
+    include ActiveModel::Conversion
+
+    def initialize(connection)
+      super(connection)
```
Comment (gabebw) [inline app/presenters/jobvite/connection_presenter.rb]: I echo Damian's question above.

File: spec/features/user_visits_dashboard_spec.rb [hunk 3]
```diff
@@ -22,10 +22,13 @@
 
       visit dashboard_path(as: user)
 
-      expect(page).to have_content t(
-        "dashboards.show.missing_namely_field",
-        name: "jobvite_id",
-      )
+      within(".jobvite-account") do
```
Comment (gabebw) [inline spec/features/user_visits_dashboard_spec.rb]: Why doesn't this scenario have a descriptive name?

Comment (hrs) [inline spec/features/user_visits_dashboard_spec.rb]: Good point. That was some other folks, but we'll fix that.

File: app/presenters/icims/connection_presenter.rb (hunk 1 above)
Comment (hrs) [inline app/presenters/icims/connection_presenter.rb]: I... don't know. I don't know.

**Pull request merged**